app/src/main/java/be/scri/services/SimpleKeyboardIME.kt

```diff
@@ -62,6 +61,9 @@ abstract class SimpleKeyboardIME(
     private var emojiBtnTablet2: Button? = null
     private var emojiSpaceTablet2: View? = null
     private var emojiBtnTablet3: Button? = null
+
+    // how quickly do we have to doubletap shift to enable permanent caps lock
```

Comment:
For comments that are their own line, @Linfye, can we capitalize the first letter and put a period at the end as they should be complete sentences? In the future :) I'll fix this!

Reply:
And also you were copying it over, so no stress, just for next time 😊